### PR TITLE
Feat tooltip

### LINKED ISSUE #89: [FEATURE] add a tooltip component
### Description

We should have a tooltip component that can be reusable anywhere. 
* it should change according to the dark-mode/light-mode
* configurable directions (Left, Right, Top, Bottom)

`open for new features`

### PR DESCRIPTION
**Pull request by VladBrok**
Closes #89

### Changes proposed

- Created a Tooltip component which has the following properties:
 1. placement - where to place the tooltip (up, down, left or right)
 2. distanceInPx - distance from the tooltip to the element that activates the tooltip on hover
 3. wrapperTag - name of the tag that wraps the tooltip (span or div)
 4. wrapperClassName - string with class names for the element that wraps the tooltip
 5. content - text of the tooltip
 6. children 
 
- Style of the tooltip can be changed inside the component

- Temporary added the tooltip in different places (Navigation and pages/index) so that it could be tested in preview deployment.

### CODE REVIEW

**Comment by rohitdasu** [comment]
@VladBrok could you please check the build issue?

**Comment by VladBrok** [comment]
> @VladBrok could you please check the build issue?

Yeah, of course! My bad, didn't noticed the build failure.

**Comment by rohitdasu** [comment]
also please resolve the conflicts @VladBrok

File: components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -59,68 +60,80 @@ export const Navbar = ({ active }: NavProps) => {
...
+              >
+                Home
+              </a>
+            </Link>
+          </Tooltip>
+          <Tooltip
+            content="Projects"
+            placement="down"
+            wrapperClassName="h-full"
+          >
```
Comment (rohitdasu) [inline components/Navbar/Navbar.tsx:81]: please remove tooltips from Home, Projects and About from the navbar.

Comment (VladBrok) [reply components/Navbar/Navbar.tsx]: Done!

**Comment by VladBrok** [comment]
> Great PR @VladBrok

@rohitdasu Thank you!

**Review by rohitdasu — APPROVED** [review]
Nice PR, Thanks for contributing 👍

**Pull request merged**